modules/lx-instance/watchmaker-lx-instance.template.cfn.yaml

```diff
@@ -867,13 +867,13 @@ Resources:
               #!/bin/bash -xe
 
               # Format and mount app volume
-              APPVOL_DEVICE="$(ls /dev/xvdf /dev/nvme1n1 2> /dev/null || true)"
-              if [[ -n "$APPVOL_DEVICE" && "${AppVolumeDevice}" == "true" ]]
+		$/dev/xvdf
+              if [[ -n "$/dev/xvdf" && "${AppVolumeDevice}" == "true" ]]
```

Comment:
be sure to test on both t2 and t3 instance types. t3 runs on the nitro system, which exposes NVMe volumes.

* https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/nvme-ebs-volumes.html
* https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-types.html#ec2-nitro-instances